### PR TITLE
Make Hold, Focus, Move actually do something

### PR DESCRIPTION
**Pull request by Teaika**
## About The Pull Request
Nobody plays as command roles because they just marines with some better guns and leadership skills. Now orders are merely doubled and their duration effect(and cooldown) have been increased, from 20 seconds duration and 45 cooldown to 1:30 duration and 2:30 cooldown.

## Why It's Good For The Game
Rebalance is always good for the game. People will start play on command roles not only for HPR and armor and mateba

## Changelog
:cl:
balance: Orders were heavily rebalanced and buffed(roughly doubled). Their duration now is 1 minute and 30 seconds, cooldown is 2 minutes and 30 seconds. Even aSL orders can now influence the battle, not mentioning FC or CO orders.
/:cl:

### CODE REVIEW

File: _maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm [hunk 1]
```diff
@@ -37099,10 +37099,10 @@
 /area/ice_colony/underground/maintenance/east)
 "bTY" = (
 /obj/machinery/atmospherics/pipe/simple/green/hidden,
+/obj/effect/decal/remains/human,
 /obj/machinery/light/small{
-	dir = 8
+	dir = 4
```
Comment (Rohesie) [inline _maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm]: Is this change intended?

**Comment by Rohesie** [comment]
The buff is a bit excessive.
3 minutes of effect for an order is a bit much. It's supposed to be a quasi-instant effect.
This system could get a refactor, but it's understandable if you don't want to do that.

We are trying to de-escalate damage and avoid powercreep. I'd warn against more marine buffs.

File: _maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm (hunk 1 above)
Comment (Teaika) [reply _maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm]: Isn't that from PR that are already merged?  Spookydonut said there's nothing more to be done to this commit

**Comment by Teaika** [comment]
Copy that, Rohesie, I'll do it when I get home. What buffs do you think I should nerf?

**Comment by Teaika** [comment]
@Rohesie Should be better now, I decreased some values and active timer is 1:30 now, cooldown is 2:30

**Comment by InterroLouis** [comment]
The speed on the move order is too high. Admittedly it was a max leadership order from the FC, but it still made me feel like I was an ancient runner, and while moving over SNOW. It's more powerful than hyperzine, and that's...a bit off. -1.2 move delay. More than twice as strong as Ancient Empress frenzy pheromones, counting the rework PR that's being TM'd. Having them last longer, though, I feel isn't as much of an issue, especially considering fights tend to last more than a few seconds. 
Also, I hate to be a buzzkill, but I'm not sure I like the wording on the new hold order shouts. Other than that, I didn't really play around with hold or aim much, used the aim your shots one once but didn't really get to see the effects.

File: _maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm (hunk 1 above)
Comment (spookydonut) [reply _maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm]: yeah this looks like an unintentional change

**Comment by spookydonut** [comment]
@Teaika need to remove the map change from this pr, ping roh or me on discord

**Comment by Teaika** [comment]
Should it be better now?

**Comment by Teaika** [comment]
I do not understand what's wrong

**Comment by InterroLouis** [comment]
Yes, it's because of the new stamina system. There are multiple references to halloss within your PR as well as several outdated files that have removed and or altered mechanics. Presence of removed defines such as xeno_armor_tackle_pen as well, and other things involving frenzy bonuses. Basically wherever I look I see things that have been changed since you made this PR and the best move at this point, in my opinion, is to rebase and do it over.

**Comment by spookydonut** [comment]
Merge conflicts again

**Comment by Rohesie** [comment]
Poke us on the coder channels to fix this kind of thing.

**Comment by Rohesie** [comment]
Closing due to staleness.
